frontend/src/app/shared/components/project-include/list/project-include-list.component.html

```diff
@@ -38,14 +38,14 @@
         >{{ project.name }}</div>
       </ng-container>
 
-      <p
+      <ul
         slot="body"
         class="spot-body-small"
       >
-        <span *ngIf="project.disabled">{{ text.does_not_match_search }}</span>
-        <span *ngIf="includeSubprojects && parentChecked">{{ text.include_all_selected }}</span>
-        <span *ngIf="project.href === currentProjectHref">{{ text.current_project }}</span>
-      </p>
+        <li *ngIf="project.disabled">{{ text.does_not_match_search }}</li>
+        <li *ngIf="includeSubprojects && parentChecked">{{ text.include_all_selected }}</li>
+        <li *ngIf="project.href === currentProjectHref">{{ text.current_project }}</li>
+      </ul>
```

Comment:
I'd only show one message in the following order:

1. This is the current project
2. Include sub projects is selected
3. The search does not match

Reply:
Hi @b12f. Agreed with giving only one reason. Can we update one the strings to the following?

1. "This is the project you are in." (no change)
2. "Project already included since Include all sub-projects is enabled." (**change**, _(text.include_all_selected)_)
3. "Project does not match the search criteria." (no change)

The priority goes from high to low, so 1 gets higher priority than 3.

FYI @marcalcobe.